spec/models/category_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "spec_helper"
+
+describe Category do
+  let(:dairy) { Category.new(name: "dairy") }
+  let(:items) { [Item.new(name: "milk", category: dairy), Item.new(name: "eggs", category: dairy)] }
```

Comment:
This line isn't being used anywhere, I guessing it's just here in preparation for more tests.
Line 17 in this file only checks for the association not for actual concrete items.